topic_modeling/scrape_training_data/get_article_links.py

```diff
@@ -5,50 +5,78 @@
 from IPython.core.display import clear_output
 from time import time
 
-# change range to 0,50 when using cluster
-pages = list(map(str, range(4)))
 
-# change range to 2012,2019 when using cluster
-year_urls = list(range(2015, 2016))
+def get_article_links(start_year, end_year, num_pages):
 
-article_links = []
+    """Collect and save a list of links that correspond to
+    articles from The Dartmouth daily newspaper.
 
-start_time = time()
-requests = 0
+    Parameters
+    ----------
+    start_year : int
+        Year in the archives to start looking for articles
+    end_year : int
+        Year in the archives to stop looking for articles
+    num_pages : int
+        Number of archive pages that fit each year range to
+        inspect
```

Comment:
isn't this something what varies from year to year and should be deduced from the pages themselves?